students/csimmons/lesson03/slice_lab.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# Craig Simmons
+# Python 210
+# slice_lab.py: Slice Lab Exercises
+# Created 11/16/2020 - csimmons
+
+# because strings are immutable, the expression in else clause
+# will not work and we need to handle them differently
+def exchange_first_last(seq):
+    if type(seq) == str:
+        first = seq[0]
+        last = seq[-1]
+        mid = seq[1:-1]
+        return (last + mid + first)
+    else:
+        seq[0],seq[-1] = seq[-1],seq[0]
+        return(seq)
```

Comment:
you should not be using index syntax - just slicing

once you slicing, you don't need to check for the data type.